<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 38, 43);</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
alternate-background-color:  &quot;#2a2c31&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/icon_back.png</normaloff>:/img/icon_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>33</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>72</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#2a2c31&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="wifiSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>540</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   热点设置</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="eqSettings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>540</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   均衡器</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>72</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot; #232429&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(47, 48, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 48, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="checkPoint">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>167</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(206, 92, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="version">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>211</y>
     <width>540</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   版本号:   v2.0</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>282</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 48, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>title</zorder>
  <zorder>back</zorder>
  <zorder>eqSettings</zorder>
  <zorder>wifiSettings</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>checkPoint</zorder>
  <zorder>version</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
